schema/schema.go

```diff
@@ -110,3 +110,50 @@ func (s *WebRPCSchema) GetServiceByName(name string) *Service {
 	}
 	return nil
 }
+
+func MatchServices(s *WebRPCSchema, services map[string]struct{}) *WebRPCSchema {
+	if s == nil {
+		return nil
+	}
+
+	if len(services) == 0 {
+		return s
+	}
+
+	var matchedServices []*Service
+	for _, srv := range s.Services {
+		if _, ok := services[srv.Name]; ok {
+			matchedServices = append(matchedServices, srv)
+		}
+	}
+
+	s.Services = matchedServices
+
+	return s
+}
+
+func IgnoreMethodsWithAnnotations(s *WebRPCSchema, annotations map[string]struct{}) *WebRPCSchema {
```

Comment:
Should this be a method on `*WebRPCSchema`?

```suggestion
// Remove methods matching the provided annotations
func (s *WebRPCSchema) RemoveMethodsWithAnnotations(annotations map[string]struct{}) {
```

Reply:
I did not want to expose these functions to `templates`.  See: https://github.com/webrpc/webrpc/blob/ca6fbe4f0e3e8948c832bc427a4ea2aacd172b44/gen/gen.go#L42

map[string] it's easier to look for specific service or annotation in map then slice :man_shrugging: